src/System.Net.HttpListener/src/System/Net/Windows/WebSockets/HttpWebSocket.Windows.cs

```diff
@@ -252,60 +203,6 @@ internal static void ValidateInnerStream(Stream innerStream)
             }
         }
 
-
-        internal static void ValidateOptions(string subProtocol,
-            int receiveBufferSize,
-            int sendBufferSize,
-            TimeSpan keepAliveInterval)
-        {
-            // We allow the subProtocol to be null. Validate if it is not null.
-            if (subProtocol != null)
-            {
-                WebSocketValidate.ValidateSubprotocol(subProtocol);
-            }
-
-            ValidateBufferSizes(receiveBufferSize, sendBufferSize);
-
-            if (keepAliveInterval < Timeout.InfiniteTimeSpan) // -1
-            {
-                throw new ArgumentOutOfRangeException(nameof(keepAliveInterval), keepAliveInterval,
-                    SR.Format(SR.net_WebSockets_ArgumentOutOfRange_TooSmall, Timeout.InfiniteTimeSpan.ToString()));
-            }
-        }
-
-        internal static void ValidateBufferSizes(int receiveBufferSize, int sendBufferSize)
-        {
-            if (receiveBufferSize < WebSocketBuffer.MinReceiveBufferSize)
```

Comment:
Looks like we're removing a bunch of references to members of WebSocketBuffer.  Can any of those members be removed, or do they all still have other references to them?